hack/verify-all.sh

```diff
@@ -37,64 +37,64 @@ res=0
 
 if [[ "${VERIFY_WHITESPACE:-true}" == "true" ]]; then
   echo "[*] Verifying whitespace..."
-  hack/verify-whitespace.sh || res=1
-  failure $res "verify-whitespace.sh"
+  hack/verify-whitespace.sh
+  failure $? "verify-whitespace.sh"
```

Comment:
hm, maybe i'm missing something. can you explain what the fix is here?
each `verify-*.sh` has a `|| res=1` which is fine and `failure()` only notifies in this case.
then near EOF `res=1/0` triggers an overall summary and the exit code matches `res`.

Reply:
ok, i see.
`$res` can leak from previously failed `verify-` calls.